app/models/payment.rb

```diff
@@ -0,0 +1,31 @@
+class Payment < ApplicationRecord
+  belongs_to :user
+  belongs_to :company, optional: true # used for branding_fee
+
+  validates_presence_of :user, :payment_type, :status
+  validates_presence_of :hips_id, on: :update
+
+  # This hash maps a HIPS order status to an SHF payment status.
+  # The payment values are stored in the DB and exposed to the user.
+  # (The user is paying a fee to SHF (payment).  In order to process that
+  #  payment, we create a HIPS order and pass that to HIPS, which then
+  #  processes that order (that is, has the user pay for the order).
+  #  Note that here, "payment" refers to the SHF Payment, and "order" to
+  #  the HIPS order).
+  # Note that succesful order payment (on the HIPS side) is represented by
+  # order status 'successful'.  On the SHF side, that translates to a
+  # completed payment ('paid') for the user fee (e.g. a membership fee).
+  ORDER_PAYMENT_STATUS = {
+    nil          => 'skapad',  # created
+    'pending'    => 'avvaktan',
+    'successful' => 'betald',   # paid
+    'expired'    => 'utgånget',
+    'awaiting_payments' => 'väntar på betalningar'
+  }.freeze
```

Comment:
Why are the SHF Payment statuses hardcoded strings?  Why not symbols?

Reply:
+    nil          => 'skapad',  # created
+    'pending'    => 'avvaktar',
+    'successful' => 'betald',   # paid
+    'expired'    => 'utgången',
+    'awaiting_payments' => 'väntar på betalning' **(is it really plural?)**

Reply:
The payment status is used in the DB and is visible to the user in views, so at this point there is not much value to using strings.

If in the future we find ourselves using one of the values in conditional logic (e.g. ```status == 'betald' ```)  then we can define a constant for that value and use it in the hash as well as in the processing code.